com.unity.render-pipelines.high-definition/Documentation~/Decal.md

```diff
@@ -0,0 +1,50 @@
+# Decal
+
+The High Definition Render Pipeline (HDRP) includes Decal Mesh and the [Decal Projector](Decal-Projector.html) component, which you can use to project specific Materials into your Scene to create realistic-looking decals. These Materials must use the [Decal Shader](Decal-Shader.md) or [Master node Decal](Master-Node-Decal.md). Use those Shader to create decal Materials that you can place, or project, into your Scene.
```

Comment:
The High Definition Render Pipeline (HDRP) includes two ways to create decals in a Scene. You can either use a decal Mesh and manually position the decal or use the [Decal Projector](Decal-Projector.html) component to project the decal. Both of these methods require that you create a decal Material, which is a Material that uses either the [Decal Shader](Decal-Shader.md) or [Decal master node](Master-Node-Decal.md). You can use either to create decal Materials that you can place or project into a Scene.